htdocs/core/modules/supplier_payment/doc/pdf_standard.modules.php

```diff
@@ -553,7 +553,11 @@ protected function _tableau_cheque(&$pdf, $object, $posy, $outputlangs)
 
 		// translate amount
 		$currency = $conf->currency;
-		$translateinletter = strtoupper(dol_convertToWord($object->montant, $outputlangs, $currency));
+		if (! empty($conf->global->MAIN_MODULE_NUMBERWORDS)) {
```

Comment:
I would prefer to have this test inside the dol_convertToWord() function.
It is goal of this function to have an abstaction layer for num to text conversion.